tests/acceptance/helpers/webUIAction.js

```diff
@@ -0,0 +1,19 @@
+const { client } = require('nightwatch-api')
+
+exports.toggleCheckbox = function (pageObject, enableOrDisable, checkboxId) {
+  client.element('css selector', checkboxId, (response) => {
+    client.elementIdSelected(response.value.ELEMENT, (result) => {
+      const checked = result.value === true
+      if ((enableOrDisable === 'disable' && checked) ||
+        (enableOrDisable === 'enable' && !checked)) {
+        // if the checkbox needs to be unchecked but it is checked
+        // Or the checkbox needs to be checked but it is unchecked
+        return pageObject
+          .click(checkboxId)
+          .click('@filterListButton')
```

Comment:
the selector might be called something else if we use the command from an other place